### DISCUSSION TITLE
OpenEFT Online Service!

### DISCUSSION THREAD

**Discussion by NoahGWood** [NoahGWood/OpenEFT#19] [Announcements]
You can finally skip the hassle of installation and use the service online at [openeft.org](https://openeft.org)!

![openeft](https://user-images.githubusercontent.com/26890744/230264173-dfe83e81-2e0c-49be-a1c0-68a41934354f.png)

Please note, this is a **paid** service, and all files are deleted within 1 hour of creation, so if you use it, be sure to download them ASAP as I cannot recover them

**Comment by Koga-Shuko** [comment]
Hi, I have been trying to use this service and after creating my card it shows none in the cart. I tried friday and today and with Chrome and Edge.

**Comment by Koga-Shuko** [comment]
So I got a little further tonight. I tried my home computer with the same result. However, when the cart showed empty I clicked add another and boom after going through it again it showed my EFT file available and I paid. Then the second I clicked the link, right after finishing the transaction, it said no file available. So it successfully liberated me of $20 and didnt do anything. Like I said a little further at least.

**Reply by NoahGWood** [reply]
Hi Koga,

Sorry to hear about that. I've identified the bug, it's in our front-end code not refreshing after the file generation completes. If you still have the success link and haven't downloaded your EFT file, you should be able to create a new file and then go to openeft.org/success?session_id=YourPaymentSession to access the new file. Unfortunately, we can't recover deleted files.

If that doesn't work for you, please reach out to me at service@openeft.org and I'll be able to assist you further. Thanks for being a customer!

**Reply by Koga-Shuko** [reply]
Got it! Thank you for the help. You all have put a lot on the front end of this and I wish I had kept up with coding. Python is all new to me. I have not coded since high school and it was basic, pascal, c++, and some java when I graduated.

**Comment by cmarti157** [comment]
Do you have a scanner that you recommend with using this program?

**Comment by av8riowa** [comment]
I cant seem to get the link to work! Please help!
